src/Config_Command.php

```diff
@@ -163,6 +163,107 @@ public function create( $_, $assoc_args ) {
 	}
 
 	/**
+	 * Define a constant in the wp-config.php file.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [<name>]
+	 * : The constant name to be defined.
+	 *
+	 * [<value>]
+	 * : The constant value to be defined.
+	 *
+	 * [--add]
+	 * : Add definition if it doesn't exist. Default: true
+	 *
+	 * [--update]
```

Comment:
`--update` has a similar issue with the `-add`.
I think we need more discussion about it.